examples/conda/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM continuumio/miniconda3:4.10.3
+
+WORKDIR /test
+RUN conda install -y conda-build
+RUN conda install -y conda-verify
+RUN conda install -y anaconda-client
```

Can we compact it into one command?
```suggestion
RUN conda install -y conda-build conda-verify anaconda-client
```